docs/source/howto/httomo_features/padding.rst

```diff
@@ -0,0 +1,31 @@
+.. default-role:: math
+.. _padding:
+
+Padding
+^^^^^^^
+
+Padding is an important feature of HTTomo when performing computations on :ref:`chunks_data` and :ref:`blocks_data`.
+If the method is a 2D method (work with 2D frames), e.g., a denoising filter, then the data does not require any padding as 
+the boundary conditions between blocks will not be violated. However, when the method is a 3D method and it works 
+with 3D volumes, then in order to satisfy boundary conditions and avoid artefacts, one need to pad blocks. 
+
+How this can be useful?
+=======================
+
+It is useful because when padding feature exists one can use true fullly 3D methods which provide a consistent resolution in
+all three dimensions, see the image bellow. Because of the access to 3D data, one can perform better in removing artefacts,
```

Comment:
Small typo:
```suggestion
all three dimensions, see the image below. Because of the access to 3D data, one can perform better in removing artefacts,
```